packages/css/src/components/Button.css

```diff
@@ -131,3 +110,28 @@
     padding: var(--space-l);
   }
 }
+
+/* Loading state */
+
+.Button-loadingState {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
Hey man, why are we using `top`, `bottom`, `right` and `left` all together in places? can we not just use `flex-grow` and `flex-shrink` (shorthand: `flex: 0 0 100%`)? Just trying understand thanks

Reply:
Hey,
It's a tricky part because when a Button transitions to a `loading` state, we don't want the button to be resized. So the loading state has to **cover** the whole button, hence the absolute positions